code/game/machinery/machinery.dm

```diff
@@ -464,10 +464,8 @@ Class Procs:
 /obj/machinery/tesla_act(var/power)
 	..()
 	if(prob(85))
-		emp_act(2)
+		explosion(src.loc,1,2,4,flame_range = 2)
```

Comment:
>should probably be the least common thing instead of 85% chance like holy crap

but then it wouldn't be the station-destroying round-ending singulo-parallel like i'm trying to make it be